data/trie/patriciaMerkleTrie.go

```diff
@@ -241,6 +242,19 @@ func (tr *patriciaMerkleTrie) Recreate(root []byte) (data.Trie, error) {
 	return newTr, nil
 }
 
+// String outputs a graphical view of the trie. Mainly used in tests/debugging
```

Comment:
Because it is mainly used in tests/debugging, wouldn't it be better to have a different printer of a tree somewhere? This way, code that is not usefull is encapsulated in a different component. This printer will also know the printfunctions for the LeafNode, BranchNode...